apps/rig_inbound_gateway/test/rig_web/presence/channel_test.exs

```diff
@@ -14,86 +14,96 @@ defmodule RigInboundGatewayWeb.Presence.ChannelTest do
   end
 
   test "a user connecting to her own topic works" do
-    assert {:ok, _response, sock} = subscribe_and_join_user(
-      "testuser",
-      [@customer_role],
-      "user:testuser"
-    )
-    leave sock
+    assert {:ok, _response, sock} =
```

Comment:
seems the changes in this file are only around reformatting and commenting the smoke tests out. If you remove this file from the PR you would spare me one merge conflict ;)